docs/src/pages/getting-started/example-projects/example-projects.md

```diff
@@ -12,6 +12,7 @@ You can find some example projects in the [GitHub repository](https://github.com
 - [Preact](https://github.com/mui-org/material-ui/tree/next/examples/preact)
 - [CDN](https://github.com/mui-org/material-ui/tree/next/examples/cdn)
 - [Plain server-side](https://github.com/mui-org/material-ui/tree/next/examples/ssr)
+- [Use styled-components as styled engine](https://github.com/mui-org/material-ui/tree/next/examples/create-react-app-with-styled-components-engine)
```

Comment:
```suggestion
- [Use styled-components as style engine](https://github.com/mui-org/material-ui/tree/next/examples/create-react-app-with-styled-components-engine)
```
?

Reply:
I used "styled engine" as the package names are @material-ui/styled-engine, mainly because they are engine for the `styled` utilities

Reply:
But for the textual description, it sounds better to use just style engine 👍